Dockerfile

```diff
@@ -1,13 +1,7 @@
 FROM node:10
 
-LABEL "com.github.actions.name"="Auto Assign"
```

Comment:
not sure if these labels are used inside the container or not but I personally kinda like these, they're descriptive (not the maintainer/repo though, good changes there)

Reply:
I thuink those are used for github actions documentation, I don't think those are used within the container